spec/lib/poseidon/consumer_group_spec.rb

```diff
@@ -136,6 +136,16 @@ def fetch_response(n)
     n.should == 400
   end
 
+  it "should accept a loop_delay" do
+    cg = described_class.new "my-group", ["localhost:29092", "localhost:29091"], ["localhost:22181"], "mytopic", logger: logger, loop_delay: 5
+    cg.loop_delay == 5
```

Comment:
this isn't a test -- to make it a test, write `cg.loop_delay.should == 5`

similarly below